app/blueprints/template/templates/create_template.html

```diff
@@ -0,0 +1,61 @@
+{% extends "base.html" %}
+{% set showNavigationBar = True %}
+{% set active_item_identifier = "templates" %}
+
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{%- from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary -%}
+{%- from "govuk_frontend_jinja/components/file-upload/macro.html" import govukFileUpload -%}
+{%- from "govuk_frontend_jinja/components/input/macro.html" import govukInput -%}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        {% if error %}
+            {{
+                    govukErrorSummary({
+                        "titleText": error.titleText,
+                        "errorList": error.errorList
+                    })
+                    }}
+        {% endif %}
+        {{ govukBackLink({
+              "text": "Back",
+              "href": url_for("template_bp.view_templates")
+            }) }}
+        <h2 class="govuk-heading-m">Add a new template</h2>
+        <div class="govuk-form-group">
+            <fieldset class="govuk-fieldset">
+                <form method="POST" enctype="multipart/form-data">
+                    {{ form.hidden_tag() }}
+                    {{ govukInput(form.template_name) }}
+                    {{ govukInput(form.tasklist_name) }}
+                    {{ govukFileUpload(form.file) }}
```

Comment:
Now that we're using govuk-frontend-wtf, we can actually just do this:
```suggestion
                    {{ form.template_name() }}
                    {{ form.tasklist_name() }}
                    {{ form.file() }}
```